scripts/export-all.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+from pathlib import Path
+
+FORMATS = [
+    "cfr",
+    "cfr-heartbeat",
+    "moments",
+    "whats-new-panel",
+    "messaging-experiments",
+]
+
+JSM_CONTENTS = """\
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// This file is generated by:
+// https://github.com/mozilla/messaging-system-inflight-assets/tree/master/scripts/export-all.py
+
+
+const EXPORTED_SYMBOLS = ["InflightAssetsMessageProvider"];
+
+const InflightAssetsMessageProvider = {{
+  getMessages() {{
+    return {messages_json};
+  }}
+}};
+"""
+
+def main():
+    all_messages = []
+    for format in FORMATS:
+        path = Path("outgoing", f"{format}.json")
+        with path.open() as f:
+            messages = json.load(f)
+
+            if messages is not None:
+                all_messages.extend(messages)
+
+    # Indent all but the first line by 4 spaces.
+    messages_json = "\n".join(
+        f"    {line}" if i > 0 else line
+        for i, line in enumerate(json.dumps(all_messages, indent=2).split("\n"))
+    )
```

Comment:
While this is a very compact way to write this, I think it would be easier to read/maintain if it were split up into a few pieces, probably with at least one intermediate variable.